### ISSUE TITLE
[20.09] Tool connection bands do not align to output

### ISSUE THREAD

**Issue by innovate-invent** [galaxyproject/galaxy#10865]
![image](https://user-images.githubusercontent.com/1768820/101419201-62e68a80-38a4-11eb-9cc1-0a478d36c3d8.png)

Moving the tool snaps the band to the output. Saving and reloading doesn't retain the corrected location.

**Comment by mvdbeek** [comment]
Do you have an error in your javascript console ?

**Comment by innovate-invent** [comment]
No errors, just `[BootstrapVue warn]: popover - Unable to find target element in document.`

**Comment by dannon** [comment]
The popover bit is almost certainly unrelated.  Are you using browser zoom, or the canvas zoom, maybe?

**Comment by innovate-invent** [comment]
neither:
![image](https://user-images.githubusercontent.com/1768820/101520999-1188da80-393a-11eb-86e5-27475b9e26fe.png)

I am running this on the most recent version of firefox

**Comment by dannon** [comment]
Good to know.  My next hunch is that it's related to those being very long output names?  Is this a pattern you've noticed?

**Comment by innovate-invent** [comment]
Good catch, yes, it appears that an output with a name greater than two lines is affected.

**Comment by dannon** [comment]
Cool -- should be able to reproduce and fix this then.  Thanks for the report!

**Comment by dannon** [comment]
Closing — the workflow editor was completely rewritten in Vue. Connection rendering now uses d3-based SVG curves (SVGConnection.vue) rather than the old band system.
